### ISSUE TITLE
Provide a Flatpak package

### ISSUE THREAD

**Issue by yosbelms** [monocasual/giada#324]
Could be nice to have a Flatpak package available on Flathub.

**Comment by monocasual** [comment]
Hi @yosbelms, thanks for your feedback. We already provide AppImage packages: adding yet another format would be quite challenging at the moment. Of course we would very happy to accept contributors!

**Comment by hfiguiere** [comment]
I have submitted in initial Flatpak to flathub.

Things that won't work: Jack - this is a limitation with sandboxing.
Things that should work: VST (only if installed through Flatpak, but the numbers are increasing)

**Comment by hfiguiere** [comment]
Also, I shall submit a PR to add the needed .desktop and appstream files, as well as install icons.

And a patch for JUCE is needed (that one isn't specific to Giada, I have it with virtually any other JUCE plugin)

**Comment by monocasual** [comment]
Cool! Just out of curiosity, why a patch for JUCE is needed?

**Comment by hfiguiere** [comment]
Yes this one. See issue #328 

tl;dr a recent g++ reject code from JUCE that is invalid. Problem isn't unique to Giada.

**Comment by hfiguiere** [comment]
It is now on flathub.

https://flathub.org/apps/details/com.giadamusic.Giada

**Comment by yosbelms** [comment]
Awesome, thank you all!

**Comment by monocasual** [comment]
Great job @hfiguiere! Just one thing: we are about to get rid of Autotools in favor of modern CMake (starting from [0.17.0](https://github.com/monocasual/giada/milestone/15)). I guess this change will destroy the current automatic packaging mechanism (if any)...

**Comment by hfiguiere** [comment]
It's not a problem. cmake is supported too.

**Comment by gvnnz** [comment]
Hey @gregorni, looks like Giada is already available as a flatpak on Flathub: https://flathub.org/apps/com.giadamusic.Giada

> You could even consider making the Flatpak the preferred packaging format for Linux.

That's an option, sure. We definitely don't want to maintain two different packages for Linux, so as soon as/if Flatpak becomes the de facto standard, we might switch to it. 🤞

**Comment by gvnnz** [comment]
Done, closing.
